### DISCUSSION TITLE
How to programatically create a new email account on a subdomain?

### DISCUSSION THREAD

**Discussion by jordanparker6** [haraka/Haraka#3137] [Q&A]
I would like to be able to create a new email account (e.g. <someid>@mydomain.com) on a HTTP request when some event happens on a backend API.

I can see the details for the address API here (https://haraka.github.io/core/Address/) 

How would I use the Haraka API to interact with this?

Would I write a plugin that listens for HTTP requests?

**Comment by msimerson** [comment]
Haraka is not a MSA. As such, there are no facilities to manage email addresses.
